qa/suites/fs/upgrade/upgraded_client/from_nautilus/tasks/2-client-sanity.yaml

```diff
@@ -1,4 +0,0 @@
-tasks:
-  - cephfs_test_runner:
-      modules:
-        - tasks.cephfs.test_newops
```

Comment:
Why are you deleting this test ? This is dedicated to test the newops based the nautilus cluster. You should add new file to test the other versions.

Reply:
Undone the removal